packages/beacon-node/src/db/buckets.ts

```diff
@@ -26,7 +26,7 @@ export enum Bucket {
   phase0_depositData = 12, // [DEPRECATED] index -> DepositData
   phase0_exit = 13, // ValidatorIndex -> VoluntaryExit
   phase0_proposerSlashing = 14, // ValidatorIndex -> ProposerSlashing
-  phase0_attesterSlashing = 15, // Root -> AttesterSlashing
+  allForks_attesterSlashing = 15, // Root -> AttesterSlashing
```

Comment:
Note: changing bucket name affects metrics label but I think we are fine with that